tests/rule-engine/fc-config.scm

```diff
@@ -1,10 +1,21 @@
 ;
 ; Configuration file for the example crisp rule base system (used by
 ; fc.scm)
-;
-; Before running any inference you must load that file
 
-; Load the rules
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Load required modules and utils ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(use-modules (opencog))
+(use-modules (opencog rule-engine))
+
+(load-from-path "utilities.scm")
+(load-from-path "av-tv.scm")
```

Comment:
Yeah, I know, I guess it must be some cruft from a time that this assumption wasn't true and I didn't dare to challenge it. If you're confident it's gonna stay true in the future I'm happy to remove these lines.